.github/workflows/test.yml

```diff
@@ -59,9 +59,9 @@ jobs:
         pip install dist/*.tar.gz
         pip freeze
 
-    - name: Check linting
-      run: |
-        make check
+    # - name: Check linting
```

Comment:
Could you add a note here that we're commenting until #102 is resolved?

Reply:
Done